documentation/content/en/books/handbook/eresources/_index.adoc

```diff
@@ -176,25 +222,23 @@ If text/plain does not accompany text/html:
 [[eresources-news]]
 == Usenet Newsgroups
 
-In addition to two FreeBSD specific newsgroups, there are many others in which FreeBSD is discussed or are otherwise relevant to FreeBSD users.
+Alongside two FreeBSD-specific groups, other groups discuss FreeBSD or are otherwise relevant to users of FreeBSD.
 
-=== BSD Specific Newsgroups
+=== BSD-oriented Groups
 
-* link:news:comp.unix.bsd.freebsd.announce[comp.unix.bsd.freebsd.announce]
-* link:news:comp.unix.bsd.freebsd.misc[comp.unix.bsd.freebsd.misc]
-* link:news:de.comp.os.unix.bsd[de.comp.os.unix.bsd] (German)
-* link:news:fr.comp.os.bsd[fr.comp.os.bsd] (French)
+* link:news:comp.unix.bsd.freebsd.announce[comp.unix.bsd.freebsd.announce] -- FreeBSD-specific
+* link:news:comp.unix.bsd.freebsd.misc[comp.unix.bsd.freebsd.misc] -- FreeBSD-specific
+* link:news:de.comp.os.unix.bsd[de.comp.os.unix.bsd] -- German
+* link:news:fr.comp.os.bsd[fr.comp.os.bsd] -- French
 
-=== Other UNIX(R) Newsgroups of Interest
+=== Other UNIX(R) Groups of Interest
 
 * link:news:comp.unix[comp.unix]
-* link:news:comp.unix.questions[comp.unix.questions]
 * link:news:comp.unix.admin[comp.unix.admin]
 * link:news:comp.unix.programmer[comp.unix.programmer]
 * link:news:comp.unix.shell[comp.unix.shell]
-* link:news:comp.unix.misc[comp.unix.misc]
 * link:news:comp.unix.bsd[comp.unix.bsd]
```

Comment:
Is it worth noting that `comp.unix` and `comp.unix.bsd` are points in a hierarchy, but not groups? 

![comp unix](https://github.com/freebsd/freebsd-doc/assets/192271/978fabcd-28e3-47c9-a834-9b304d1ffcaa)